doc/user_guide/signal/signal_basics.rst

```diff
@@ -133,111 +129,149 @@ example we create a Signal2D instance from a 2D numpy array:
+    ``EleXSpy`` and the subclass ``hologram`` has been
+    moved to the extension package ``HoloSpy``.
+
+.. _hyperspy_extensions-label:
+
+HyperSpy extensions
+^^^^^^^^^^^^^^^^^^^
+
+Domain specific functionalities for specific types of data are provided through
+a number of dedicated python packages that qualify as `HyperSpy extensions`. These
+packages provide subclasses of the generic signal classes listed above, depending
+on the dimensionality and type of the data. Some examples are included in the
+:ref:`diagram above <signal_overview_figure-label>`.
+If an extension package is installed on your system, the provided signal
+subclasses are registered with HyperSpy and these classes are directly
+available when loading the ``hyperspy.api`` into the namespace. A `list of packages
+that extend HyperSpy <https://github.com/hyperspy/hyperspy-extensions-list>`_
+is curated in a dedicated repository.
+
+The metadata attribute ``signal_type`` describes the nature of the signal. It can
+be any string, normally the acronym associated with a particular signal. To print
+all :py:class:`~.api.signals.BaseSignal` subclasses available in your system call
+the function :py:func:`~.api.print_known_signal_types` as in the following
+example:
 
 .. code-block:: python
 
     >>> hs.print_known_signal_types()
     +--------------------+---------------------+--------------------+----------+
     |    signal_type     |       aliases       |     class name     | package  |
     +--------------------+---------------------+--------------------+----------+
-    | DielectricFunction | dielectric function | DielectricFunction | hyperspy |
-    |      EDS_SEM       |                     |   EDSSEMSpectrum   | hyperspy |
-    |      EDS_TEM       |                     |   EDSTEMSpectrum   | hyperspy |
-    |        EELS        |       TEM EELS      |    EELSSpectrum    | hyperspy |
-    |      hologram      |                     |   HologramImage    | hyperspy |
-    |      MySignal      |                     |      MySignal      | hspy_ext |
+    | DielectricFunction | dielectric function | DielectricFunction | elexspy  |
+    |      EDS_SEM       |                     |   EDSSEMSpectrum   | elexspy  |
+    |      EDS_TEM       |                     |   EDSTEMSpectrum   | elexspy  |
+    |        EELS        |       TEM EELS      |    EELSSpectrum    | elexspy  |
+    |      hologram      |                     |   HologramImage    | holospy  |
     +--------------------+---------------------+--------------------+----------+
 
-.. warning::
-    From version 2.0 HyperSpy will no longer ship
-    :py:class:`~.api.signals.BaseSignal` subclasses that are specific to a
-    particular type of data (i.e. with non-empty ``signal_type``). All those
-    signals currently distributed with HyperSpy will be moved to new
-    packages.
+Object oriented functions provided by the extensions will be available after
```

Comment:
This information is not clear to me. Which functions from extensions are available after `import hyperspy.api as hs`? Do you mean that an extension signal loaded with `s = hs.load("signal.h5", signal_type="EBSD")` will have additional class methods not provided by HyperSpy?

Reply:
Yes, for example for an EBSD signal, you provide the `s.extract_grid()` method. I'll try to rephrase it to make it more clear.

Reply:
Thanks for updating. Perhaps the final sentence in that paragraph could be a recommendation to visit the extensions' references for lists of these functions and methods?